.github/CONTRIBUTING.md

```diff
@@ -31,70 +31,93 @@ If you at any point have any questions please read: [I have a question related t
   - [UX review](#ux-review)
   - [Self review](#self-review)
 
-## The contribution model
-<!-- This section has the purpose of explaining why we do it like we do, in order to have people 
-symphatize with the process -->
-- What are we trying to achieve with this contribution model
-  - Cooperation! - UXers, tech people, and contributors should all be contributing to make the best possible design system
-  - Issues for trackability and history
-  - Why do we do Tech refinement?
-  - Why we do UX refinement?
-  - Why we do UX reviews?
-  - Why do a self-review?
-- General description of the contribution model
+## Introduction to the contribution model
+<!-- Introduction and motivation -->
+Despite Kirby stemming from an internal Bank Data project - it has been decided to open source it, such that others than team Kirby can contribute to it. 
+Allowing for Kirby to be used in multiple projects without team Kirby being the bottleneck. 
+
+If you have a problem, an issue you want resolved, or even got brand new ideas for components. Well. There is no need to sit around and wait for team Kirby to get it implemented. 
+You can contribute yourself! And no matter your role, you can probably contribute with something. May it be a helpful comment or a bug report.
+
+We think that is neat. 
+
+There is still need for managing Kirby however. We need to make sure that the project is moving in the right direction. That submitted code is: high quality, not duplicating anything and consistent with the rest of the codebase.
+To do so a series of steps has been created that an issue must go through. All of these aim at making sure that the process of cooperating from idea to implementation is smooth for contributors. With as little time wasted as possible.
+
+<!-- General description of the contribution model -->
+In broad strokes the process is: submit an issue, get it refined, implement it, get it reviewed. Once reviewed it will automatically be merged to master. 
```

Comment:
Perhaps we should use the word "approve" in this sentence, e.g.

> In broad strokes the process is: submit an issue, get it refined, implement it, get it reviewed and approved. Once approved it will automatically be merged to master.

Also, consider to use an ordered list:

1. Submit an issue
2. Get it refined
3. Implement it
4. Get it reviewed and approved

Reply:
Should be solved by new commits ☀️